styleguide/source/assets/scss/00-base/_layout.scss

```diff
@@ -17,3 +17,18 @@
     display: table;
   }
 }
+
+//Adds flex properties
+.ama__flex-container {
```

Comment:
Drupal adds DIVs which breaks flexbox. This class adds flexbox back to new DIVs introduced by Drupal/